src/ConnectedMode/UI/ConnectedModeManager.cs

```diff
@@ -0,0 +1,53 @@
+﻿/*
```

Comment:
The ConnectedModeManager class should no longer exist, but it's probably here due to merge conflicts. :( It was renamed to ConnectedModeUIManager. We have to drop this class.